moveit_core/online_signal_smoothing/src/butterworth_filter.cpp

```diff
@@ -96,12 +96,12 @@ bool ButterworthFilterPlugin::initialize(rclcpp::Node::SharedPtr node, moveit::c
 {
   node_ = node;
   num_joints_ = num_joints;
+  auto param_listener = std::make_unique<online_signal_smoothing::ParamListener>(node_);
+  auto filter_coeff = param_listener->get_params().butterworth_filter_coeff;
```

Comment:
This doesn't need to be a unique_ptr nor does it need to stay alive beyond the initial reading of the params. You can limit the scope with some brackets:

```suggestion
  {
    online_signal_smoothing::ParamListener param_listener(node_);
    auto filter_coeff = param_listener.get_params().butterworth_filter_coeff;
  }
```